src/http/web_socket/close_codes.cr

```diff
@@ -0,0 +1,14 @@
+module HTTP::WebSocket::CloseCodes
+  NormalClosure           = 1000
+  GoingAway               = 1001
+  ProtocolError           = 1002
+  UnsupportedData         = 1003
+  NoStatusReceived        = 1005
+  AbnormalClosure         = 1006
+  InvalidFramePayloadData = 1007
+  PolicyViolation         = 1008
+  MessageTooBig           = 1009
+  MandatoryExtension      = 1010
+  InternalServerError     = 1011
+  TLSHandshake            = 1015
```

Comment:
These look more like an enum members, constants are usually defined as `SNAKE_CASED_WITH_ALL_CAPS`.

Reply:
This is intentional.

Reply:
And what's the intention behind it?

Reply:
to look like an enum

Reply:
So why not make one (it's extensible, so there would be no need for an Union) instead of this?

Reply:
code is art, everything is subjective, nothing matters, let it go